tests/gui/test_commands.py

```
@@ -1046,3 +1048,65 @@ def on_triggered():
+    assert context.state["frame_idx"] == skipped_to_frame
+    assert len(labels.find(video, skipped_to_frame)[0].instances) != initial_count
+
+
+def test_go_prev_instance_change(centered_pair_predictions: Labels):
+    """Test that goto next instance change command works correctly."""
+    # Set up test data
+    labels = centered_pair_predictions
+    video = labels.videos[0]
+
+    # Create frames with different numbers of instances
+    lf1 = context.labels.find(video, video.num_frames - 1, return_new=True)[
```

Ruff (0.8.2)

Undefined name `context`

(F821)